core/components/organisms/pageHeader/__stories__/withFilter.story.jsx

```diff
@@ -31,13 +32,21 @@ export const level0WithFilter = () => {
   );
 
   return (
-    <div className="py-6 bg-secondary-lightest" style={{ width: '900px' }}>
+    <div className="py-6 bg-secondary-lightest Pageheader-wrapper">
```

Comment:
Please remove the `Pageheader-wrapper` class. This class does not make much difference or can we please use `Col--11` class